nectar/downloaders/threaded.py

```diff
@@ -257,7 +256,9 @@ def _fetch(self, request, session):
             )
             report.download_skipped()
 
-        except requests.ConnectionError:
+        except requests.ConnectionError, e:
```

Comment:
I think the `as` syntax may be better.

`except requests.ConnectionError as e:`